omero/sysadmins/version-requirements.txt

```diff
@@ -1229,9 +1229,9 @@ Ice
       - from June 2015
       - to TBA
       - Unsupported
-      - Unsupported
+      - Supported
       - Recommended
-      - :zerocforum:`Ice 3.6.0 <announcements/6631-ice-3-6-0-and-ice-touch-3-6-0-released>`
```

Comment:
Why do you want to keep Ice 3.6.0 since we cannot use it? i.e. it cannot be a supported version